fe2-android/app/src/main/res/layout/fragment_lao_detail.xml

```diff
@@ -1,141 +1,141 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto">
-
-    <data>
-        <import type="android.view.View" />
-        <variable
-            name="view_model"
-            type="com.github.dedis.student20_pop.detail.LaoDetailViewModel"/>
-    </data>
-
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/fragment_lao_detail"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+    <import type="android.view.View" />
+    <variable
+      name="view_model"
+      type="com.github.dedis.student20_pop.detail.LaoDetailViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/fragment_lao_detail"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include layout="@layout/tab_properties" />
+
+    <androidx.constraintlayout.widget.Guideline
+      android:id="@+id/guideline_horizontal_tab"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:orientation="horizontal"
+      app:layout_constraintGuide_percent="@dimen/guideline_horizontal_tab" />
+
+    <LinearLayout
+      android:id="@+id/lao_detail_linear_layout"
+      android:layout_width="match_parent"
+      android:layout_height="0dp"
+      android:orientation="vertical"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/guideline_horizontal_tab">
+
+      <LinearLayout
+        android:id="@+id/properties_linear_layout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-        <include layout="@layout/tab_properties" />
-
-        <androidx.constraintlayout.widget.Guideline
-            android:id="@+id/guideline_horizontal_tab"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:orientation="horizontal"
-            app:layout_constraintGuide_percent="@dimen/guideline_horizontal_tab"/>
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:visibility="visible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/guideline_horizontal_tab">
+
+        <ImageButton
+          android:id="@+id/edit_button"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          android:src="@drawable/ic_baseline_edit_24"
+          android:visibility="visible"
```

Comment:
This edit button should only be visible when the user is an organiser, because only the organiser can change the name of the LAO